client/sttp-client4/src/main/scala/sttp/tapir/client/sttp4/SttpClientInterpreter.scala

```diff
@@ -0,0 +1,19 @@
+package sttp.tapir.client.sttp4
+
+import sttp.tapir.{DecodeResult, Endpoint, PublicEndpoint}
+
+private[sttp4] trait SttpClientInterpreter {
+  final protected def throwDecodeFailures[T](dr: DecodeResult[T]): T =
```

Comment:
to avoid extending traits to pull in utility methods, these can probably be moved to a package object and made package-private

Reply:
done